src/main/java/careneighbors/community/Image/Image.java

```diff
@@ -0,0 +1,37 @@
+package careneighbors.community.Image;
+
+
+import careneighbors.community.post.Post;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Entity
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@ToString
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class Image {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
```suggestion
@EqualsAndHashCode(onlyExplicitlyIncluded = true)
public class Image {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @EqualsAndHashCode.Include
    private Long id;
```

@EqualsAndHashCode(onlyExplicitlyIncluded = true) 를 사용하는 이유는 임의의 필드들만 Equals함수로 비교하기 위해서 입니다. 비교하려는 필드(엔티티의 경우에는 ID필드)에 @EqualsAndHashCode.Include를 붙여주셔야합니다